projects/packages/forms/src/dashboard/integrations/style.scss

```diff
@@ -12,7 +12,8 @@
 		display: flex;
 		flex-direction: column;
 		gap: 8px;
-		margin-bottom: 24px;
+		margin: 0 auto 24px auto;
+		max-width: 620px;
```

Comment:
From what I can tell, the new CSS here is only applying to the header of the page (ie, Streamline your forms...) and not the full integrations panel. I assume we want the max width on the full integrations panel. If so, we need to bump the max width from `.jp-forms__integrations-header` to the full `.jp-forms__integrations` container, and likely adjust some padding. I can push that change shortly. I'm playing with the widths and paddings locally.

Reply:
Sounds good!